### PR TITLE
Localized file check-in by OneLocBuild Task: Build definition ID 981: Build ID 1445118

### PR DESCRIPTION
**Pull request by MiYanni**
This is the pull request automatically created by the OneLocBuild task in the build process to check-in localized files generated based upon translation source files (.lcl files) handed-back from the downstream localization pipeline. If there are issues in translations, visit https://aka.ms/ceLocBug and log bugs for fixes. The OneLocBuild wiki is https://aka.ms/onelocbuild and the localization process in general is documented at https://aka.ms/AllAboutLoc.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/project-system/pull/7730)

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/xlf/Folder.xaml.cs.xlf [hunk 1]
```diff
@@ -1,42 +1,42 @@
...
-      </trans-unit>
-    </body>
-  </file>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
+  <file datatype="xml" source-language="en" target-language="cs" original="../Folder.xaml">
+    <body>
+      <trans-unit id="Category|Misc|DisplayName">
+        <source>Misc</source>
+        <target state="new">Misc</target>
```
[the remark this answers is not in the document]
Comment (MiYanni) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/xlf/Folder.xaml.cs.xlf:7]: The translations have already been processed via https://github.com/dotnet/project-system/pull/7732. So, this isn't a concern to merge. Just means that there will be a followup PR to this one that fixes Misc again.

**Comment by MiYanni** [comment]
My belief is that merging https://github.com/dotnet/project-system/pull/7732 first will fix this build since the Spanish build seems to be failing because of translation issues.

**Comment by MiYanni** [comment]
I don't know how to fix this without further investigation.

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/xlf/BuildPropertyPage.xaml.zh-Hans.xlf [hunk 2]
```diff
@@ -189,12 +189,12 @@
...
       <trans-unit id="DynamicEnumProperty|PlatformTarget|Description">
         <source>Specifies the processor to be targeted by the output file. Choose 'Any CPU' to specify that any processor is acceptable, allowing the application to run on the broadest range of hardware.</source>
-        <target state="new">Specifies the processor to be targeted by the output file. Choose 'Any CPU' to specify that any processor is acceptable, allowing the application to run on the broadest range of hardware.</target>
+        <target state="translated">指定输出文件的目标处理器。选择“任何 CPU”以指定任何处理器都是可接受的，从而允许应用程序在最广泛范围的硬件上运行。</target>
         <note />
       </trans-unit>
       <trans-unit id="DynamicEnumProperty|PlatformTarget|DisplayName">
         <source>Platform target</source>
-        <target state="new">Platform target</target>
+        <target state="translated">平台目标</target>
```
Comment (Varorbc) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/xlf/BuildPropertyPage.xaml.zh-Hans.xlf:197]: It is more appropriate to translate to the '目标平台', and it is the same in the legacy UI

Comment (MiYanni) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/xlf/BuildPropertyPage.xaml.zh-Hans.xlf:197]: I can make a translation issue for it. Thanks!

**Review by tmeschter — APPROVED** [review]

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/xlf/BuildPropertyPage.xaml.zh-Hans.xlf (hunk 2 above)
Comment (MiYanni) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/xlf/BuildPropertyPage.xaml.zh-Hans.xlf:197]: I've created an issue for it. It is internally tracked, but we should see the change in a future translation PR. Thanks!
https://ceapex.visualstudio.com/CEINTL/_workitems/edit/514811

**Pull request merged**